torchgeo/trainers/__init__.py

```diff
@@ -11,13 +12,16 @@
 from .resisc45 import RESISC45ClassificationTask, RESISC45DataModule
 from .sen12ms import SEN12MSDataModule, SEN12MSSegmentationTask
 from .so2sat import So2SatClassificationTask, So2SatDataModule
-from .tasks import ClassificationTask
+from .tasks import ClassificationTask, MultiLabelClassificationTask
 from .ucmerced import UCMercedClassificationTask, UCMercedDataModule
 
 __all__ = (
     # Tasks
     "ClassificationTask",
+    "MultiLabelClassificationTask",
     # Trainers
+    "BigEarthNetClassificationTask",
+    "BigEarthNetDataModule",
     "BYOLTask",
```

Comment:
Many of these are also Tasks. In my mind, there are two types of Trainers: Tasks and DataModules. Or does PyTorch-lightning have some different meaning than what I'm thinking of?

Reply:
No you're right. The Tasks are LightningModules. DataModules I wouldn't consider tasks but they are grouped in the same script with the tasks. I'll break them out into tasks and datamodules.